### DISCUSSION TITLE
Updating firmware CC2652R without repairing

### DISCUSSION THREAD

**Discussion by atmel14** [Koenkk/zigbee2mqtt#8532] [General]
Hello.
I have update my zigbee2mqtt to 1.21.0 and is run with my slae.sh CC2652R stick and 27 devices.
But I have see there is also a new firmeware for CC2652R available.
https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

How is the right process to flash new firmeware without repairing my devices?

Thanks in advance.
BYe
Thomas

**Comment by Nardol** [comment]
I have updated my CC2652R (ZZH) and did not have to repair my devices thanks to the coordinator backup created by Zigbee2MQTT.
Updating coordinator does not require repairing: https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices
So you can follow regular update method for your stick explained on this page: https://slae.sh/projects/cc2652/

**Comment by Nardol** [comment]
Restart Zigbee2MQTT one time to be sure to have a full coordinator backup: https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.21.0

**Reply by theetron** [reply]
I'm using Home Assistant wit this stick and did the upgrade. To restart Zigbee2mqtt, use the restart button in de zigbe2mqtt GUI under Settings/Tools. A restart in the supervisor isn't creating a backup (when I did this a while ago).

**Comment by atmel14** [comment]
Hello.

Many thanks.
I have now two CC2652R. 
Is it also possible to change the hardware (use the other usb stick) without repairing?

BYE

**Reply by Koenkk** [reply]
yes
